### PR TITLE
add : vivek-kumar-singh

### PR DESCRIPTION
**Pull request by vivekthedev**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1080"></a>

### CODE REVIEW

**Review by syedsohan — APPROVED** [review]

File: public/data/vivek.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vivekthedev",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Won Hackathon",
+      "date": "October 2021",
+      "icon": "Globe",
```
Comment (ssensalo) [inline public/data/vivek.json:22]:
```suggestion
      "icon": "globe",
```

File: public/data/vivek.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      "color": "grey",
+      "description": "Won prixe for notable mentions"
```
Comment (ssensalo) [inline public/data/vivek.json:24]:
```suggestion
      "description": "Won prize for notable mentions"
```

**Review by ssensalo — CHANGES_REQUESTED** [review]

File: public/data/vivek.json (hunk 1 above)
Comment (Panquesito7) [reply public/data/vivek.json:22]: This can be kept as-is, it's fine. 🙂

**Review by Panquesito7 — APPROVED** [review]

File: public/data/vivek.json (hunk 1 above)
Comment (KendallDoesCoding) [reply public/data/vivek.json:22]: @Panquesito7 Yes, but for formatting purposes, it's better in lowercase, coz the rest of the icons are in lower-case

**Review by KendallDoesCoding — APPROVED** [review]
LGTM

**Review by vikasganiga05 — APPROVED** [review]
LGTM :+1:

**Review by ssensalo — APPROVED** [review]

**Pull request merged**